rtic_v5/bluepill_blinky/README.md

```diff
@@ -35,63 +35,13 @@ You need to connect you bluepill board to ST-Link and connect pins:
 
 Plug in ST-Link to USB port and wait it to initialize.
 
-### Upload
+### Flashing and running
 
-We will use openocd to upload the code wit simple one-line one-command script:
+Flashing with a standard STLink v2 is easy with `cargo-embed`:
```

Comment:
Why even mention the STLink? Any probe that is supported by probe-rs can be used.

Reply:
Sorry, I missed these comments. Could you transfer this to a new PR?